src/watertap_contrib/reflo/unit_models/air_stripping_0D.py

```diff
@@ -0,0 +1,1221 @@
+        ),
+    )
+
+    def build(self):
+        super().build()
+
+        target = self.config.target
+        self.target_set = Set(initialize=[target])
+        comps = self.config.property_package.component_list
+        solutes = self.config.property_package.solute_set
+        phase_set = self.config.property_package.phase_list
+        self.liq_target_set = Set(initialize=["Liq"]) * self.target_set
+        phase_target_idx = list(itertools.product(["Liq", "Vap"], self.target_set))
+        self.phase_target_set = Set(initialize=phase_target_idx)
+
+        self.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+        self.process_flow = ControlVolume0DBlock(
+            dynamic=False,
+            has_holdup=False,
+            property_package=self.config.property_package,
+            property_package_args=self.config.property_package_args,
+        )
+
+        self.process_flow.add_state_blocks(has_phase_equilibrium=True)
+        self.process_flow.add_material_balances(
+            balance_type=self.config.material_balance_type, has_mass_transfer=True
+        )
+        self.process_flow.add_energy_balances(
+            balance_type=self.config.energy_balance_type, has_enthalpy_transfer=False
+        )
+
+        self.process_flow.add_momentum_balances(
+            balance_type=self.config.momentum_balance_type,
+            has_pressure_change=True,
+        )
+
+        prop_in = self.process_flow.properties_in[0]
+        prop_out = self.process_flow.properties_out[0]
+
+        self.add_inlet_port(name="inlet", block=self.process_flow)
+        self.add_outlet_port(name="outlet", block=self.process_flow)
+
+        self.process_flow.mass_transfer_term[0, "Liq", "H2O"].fix(0)
+        self.process_flow.mass_transfer_term[0, "Vap", "Air"].fix(0)
+
+        self.air_water_ratio_param = Param(
+            initialize=3.5,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Factor multiplied by minimum air-to-water flow ratio for operating air-to-water flow ratio",
+        )
+
+        self.pressure_drop_tower_param = Param(
+            initialize=275,
+            mutable=True,
+            units=(pyunits.newton * pyunits.s**2) / pyunits.m**4,
+            doc="Factor to calculate pressure drop through tower",
+        )
+
+        self.tower_height_factor = Param(
+            initialize=1.2,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Factor to calculate tower height",
+        )
+
+        self.tower_port_diameter = Param(
+            initialize=6,
+            mutable=True,
+            units=pyunits.inches,
+            doc="Diameter of tower access ports: 2-24 inches",
+        )
+
+        self.tower_pipe_diameter = Param(
+            initialize=6,
+            mutable=True,
+            units=pyunits.inches,
+            doc="Diameter of tower inlet and outlet piping: 2-24 inches",
+        )
+
```

Comment:
Since you state the diameters for tower_port and tower_pipe in the doc, should bounds be added that reflect that? bound=(2,24)

Reply:
I would but you can't add bounds to a ``Param``